resources/v2/03_bookings.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE bookings (
+    id INTEGER NOT NULL PRIMARY KEY,
+    pickup TEXT NOT NULL,
+    destination TEXT NOT NULL,
```

Comment:
It should hold location coordinates. Use Postgis extension and set it geometry type  http://postgis.net/workshops/postgis-intro/geometries.html

Reply:
Yes, created the postgis extension & used the geometry type.